test-suite/testkit/src/builder.rs

```diff
@@ -198,6 +160,9 @@ impl TestKitBuilder {
     }
 
     /// Adds a node plugin to the testkit.
+    ///
+    /// This method is only available if the crate is compiled with the `exonum-node` feature,
```

Comment:
It seems that since `exonum-node` feature is not enabled by default, this method will not appear on `docs.rs` (at least it does not in local `cargo doc` build).
Maybe we should make this feature a default one, and just specify `default-features = false` in packages that do not need it?